packages/editor/src/client/blocks/teaserGridBlock.tsx

```diff
@@ -203,13 +207,14 @@ export function TeaserBlock({
 }
 
 export function contentForTeaser(teaser: Teaser, numColumns: number) {
+  const {t} = useTranslation()
   switch (teaser.type) {
     case TeaserType.Article: {
       const states = []
 
-      if (teaser?.article?.draft) states.push('Draft')
-      if (teaser?.article?.pending) states.push('Pending')
-      if (teaser?.article?.published) states.push('Published')
+      if (teaser?.article?.draft) states.push(t('draft'))
+      if (teaser?.article?.pending) states.push(t('pending'))
+      if (teaser?.article?.published) states.push(t('published'))
```

Comment:
Please use a more descriptive translation key for the three states. Something like `state_draft`